Neos.Cache/Classes/Backend/PdoBackend.php

```diff
@@ -518,6 +518,12 @@ public function valid(): bool
      */
     public function rewind()
     {
+        try {
+            $this->connect();
+        } catch (Exception $e) {
+            return;
+        }
```

Comment:
Is that related?

Reply:
ah, you mention it in the PR description..
I always struggle a bit when I see a `catch` block that is ignored.

Reply:
Should maybe have been a separate PR.

Reply:
Alternative would be to override getIterator and call connect there.